README.md

```diff
@@ -4,6 +4,9 @@
 
 Fast, concise, zero fun resolver for ember apps.
 
+Removes a bunch of stuff like pod support, bunch of a fallbacks, a bunch of different normalization’s etc.
```

Comment:
TODO: We should likely improve the actual wording here, and make a list.

Reply:
cc @gabrielcsapo

Reply:
I agree, I think it would be good to also move this under a section called "Motivation".

Reply:
Would you like to close this out and add the wording that works for you? I'd have to spend some time comparing the two projects to come up with something good. The main purpose of submitting this PR was to get it started as I couldn't tell at first glance! (also happy to close this out if it's too soon to be worrying about docs, don't mean to add a bunch of work for you guys :)